lib/fizzy/parser_expr.cpp

```diff
@@ -39,6 +39,7 @@ parser_result<uint8_t> parse_blocktype(const uint8_t* pos, const uint8_t* end)
     if (pos == end)
         throw parser_error{"Unexpected EOF"};
 
+    // TODO: use parse_byte here?
```

Comment:
Yes.

Reply:
I would say actually this one cannot be nicely changed because `validate_valtype` also wants to read it.

Reply:
Change to `void validate_valtype(uint8_t)`.

Reply:
`valida_valtype` is just a wrapper over `parse<ValType>`

Reply:
Done.